qwandaq/src/main/java/life/genny/qwandaq/entity/PCM.java

```diff
@@ -36,6 +36,10 @@ public class PCM extends BaseEntity {
 	public static final String PCM_DETAIL_VIEW = "PCM_DETAIL_VIEW";
 	public static final String PCM_PROCESS = "PCM_PROCESS";
 	public static final String PCM_EVENTS = "PCM_EVENTS";
+	public static final String PRI_LOC1 = "PRI_LOC1";
```

Comment:
Please remove these location constants. we should be using this instead:
```java
PCM.location(1);
```

Reply:
Used PCM.location